docs/install_Device_Northbound.md

```diff
@@ -44,96 +44,94 @@ Instead of manually configuring you can also import the configuration files:
 
 2. Click on "Integrate"
    
-  ![IIH_IntegrateDataService](graphics/IIH_IntegrateDataService.png)
+  ![IIH_Integrate](graphics/IIH1.png)
 
 3. Confirm the integration in the popup window
 
-  ![IIH_IntegrateDataServiceConfirmation](graphics/IIH_IntegrateDataServiceConfirmation.png)
+  ![IIH_IntegrateConfirmation](graphics/IIH2.png)
+
+### Configure Connection to Insights Hub.
 
-### Configure Connections
+Common Configurator needs a connection to the Databus and to Insights Hub. 
 
-The IIH needs a connection to the Databus and to Insights Hub. You need to have a Insights Hub account and create
-certificates to allow a connection from the IIH.
+You need to have a Insights Hub account and create certificates to allow a connection from the Common Configurator.
 
 1. Configure the Databus credentials under "Settings > Databus credentials"
 
-  ![IIH_DatabusCredentials](graphics/IIH_DatabusCredentials.png)
+   ![IIH_DatabusCredentials](graphics/IIH3.png)
 
 2. Save the configuration
 
-3. Check if the device has an online connection in the "Home" tab, to proceed with configuring the Insights Hub connection.
-
-  ![IIH_OnlineStatus](graphics/IIH_OnlineStatus.png)
+3. Proceed with configuring the Insights Hub connection by clicking "Add" button.
 
-4. Click on "Add parent IED / Connect to MindSphere"
+   ![IIH_OnlineStatus](graphics/IIH4.png)
 
-  ![IIH_MindsphereConfig1](graphics/IIH_MindsphereConfig1.png)
-
-5. Select the device type "MindSphere Device"
-
-6. Enter credentials for the application. Those are required, to allow the IIH to interact with Insights Hubs REST-API.
-   The Insights Hub Tenant Administrator has to create them and assign the role "mdsp:core:Admin3rdPartyTechUser", to
-   allow the IIH to update the asset model in Insights Hub. More information can be found in the [IIH documentation](https://cache.industry.siemens.com/dl/dl-media/582/109803582/att_1087779/v6/EdgeApp_CommonConfigurator_en-US/en-US/index.html#treeId=487dc5ea471ae3fbdfb56d715301dad7) and [here](https://documentation.mindsphere.io/MindSphere/apps/operator-cockpit/application-credentials-for-API-applications.html) 
+4. Select the Target type. Enter credentials for the application. Those are required, to allow the IIH to interact with Insights Hubs REST-API.
+   
+   ![IIH_InsightsHub](graphics/IIH5.png)
 
-  ![IIH_MindsphereConfig2](graphics/IIH_MindsphereConfig2.png)
+   The Insights Hub Tenant Administrator has to create them and assign the role "mdsp:core:Admin3rdPartyTechUser", to allow the IIH to update the asset model in Insights Hub. More information can be found in the [IIH documentation](https://cache.industry.siemens.com/dl/dl-media/582/109803582/att_1087779/v6/EdgeApp_CommonConfigurator_en-US/en-US/index.html#treeId=487dc5ea471ae3fbdfb56d715301dad7) and [here](https://documentation.mindsphere.io/MindSphere/apps/operator-cockpit/application-credentials-for-API-applications.html)
```

Comment:
I would recommend not using Application Credentials with Admin3rdPartyTechUser but create technical user in Settings of Insights Hub as this is now the prefered method to authenticate IIH against REST API. Currently not part of Doku